cluster/gce/trusty/configure.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
Thanks. I notice Yifan is refactoring and adding coreos supports. I would like to coordinate with Yifan for the refactoring effort, so as to avoid conflict and to make stuff under cluster/gce/ cleaner and conciser.

@yifan-gu , does it make sense?